padertorch/contrib/examples/source_separation/or_pit/train.py

```diff
@@ -163,7 +173,8 @@ def prepare_iterable(
     if iterator_slice is not None:
         iterator = iterator[iterator_slice]
 
-    # This
+    # Group iterators by number of speakers so that all examples in a batch
+    # have the same number of speakers
     iterators = list(iterator.groupby(lambda x: x['num_speakers']).values())
 
     iterators = [
```

Comment:
```python
if prefetch:
    iterator = iterator.prefetch(8, 16, catch_filter_exception=True)
elif chunk_size > 0:
    iterator = iterator.catch()
```
Can you add a comment, why chunk_size >= 0 cannot raise an FilterException?

Reply:
Done